doc/offline-signing-tutorial.md

```diff
@@ -0,0 +1,259 @@
+```
+
+### Decode and Analyze the PSBT Transaction
+
+1. Decode and analyze the PSBT transaction on the `offline_wallet` using the `funded_psbt.txt` file.
+
+```sh
+[offline]$ ./src/bitcoin-cli -signet decodepsbt $(cat ~/funded_psbt.txt)
+{
+    ...
+}
+[offline]$ ./src/bitcoin-cli -signet analyzepsbt $(cat ~/funded_psbt.txt)
+{
+  "inputs": [
+    {
+      "has_utxo": true,
+      "is_final": false,
+      "next": "signer",
+      "missing": {
+        "signatures": [
+          "d1825532924fff9704a8e6cbb403e88259948c43"
+        ]
+      }
+    }
+  ],
+  "estimated_vsize": 141,
+  "estimated_feerate": 0.00001000,
+  "fee": 0.00000141,
+  "next": "signer"
+}
+```
+
+### Process and Sign the PSBT Transaction
+
+1. Process and sign the PSBT transaction on the `offline_wallet`.
+
+```sh
+[offline]$ processed_psbt=$(./src/bitcoin-cli -signet -rpcwallet="offline_wallet" walletprocesspsbt $(cat ~/funded_psbt.txt) | jq -r .psbt)
+[offline]$ echo $processed_psbt
+cHNidP8BAHECAAAAAWXoonGoVULy7an4Z+TcQAAuT0axtGZAcS/Qp3AQ2QgEAQAAAAD9////AoA4AQAAAAAAFgAULajnzvO5M38eEwmu9dF+xH5m5RGDJgAAAAAAABYAFMmkmFUvb9NZrPu+VxeaMLiwdbHHAAAAAAABAHECAAAAAefmKOqzoEKbtyEsrdzdiOId8oXPcNM0OBHxeNPfRdHgAAAAAAD9////AiPiDQAAAAAAFgAUZJLFRE3DQ9BMSF62gBYs69jH9A+QXwEAAAAAABYAFNGCVTKST/+XBKjmy7QD6IJZlIxDcGsCAAEBH5BfAQAAAAAAFgAU0YJVMpJP/5cEqObLtAPoglmUjEMBCGsCRzBEAiBlnlRPN9G0iSPfid/3HGPZlsCyMxg8WPhKdz2OYwbloAIgdly1oxQDMgzkmrZtb6CXpwsAoTGr+oMlJLzIQK8lH2cBIQK2YwOmnWYOmMSTIK/Cjdmr2PNMnVnEyiFIokP6gR6myAAAIgICHIVf9O/R/hXdHHtxHTPNHgDBbMZGqVka2lGeedAWuysYQTA3wVQAAIABAACAAAAAgAEAAAAAAAAAAA==
+```
+
+### Finalize the PSBT Transaction
+
+1. Finalize the PSBT transaction on the `offline_wallet` and save the final PSBT to the `final_psbt.txt` file, to be exported and broadcasted by the online wallet
+
+```sh
+[offline]$ ./src/bitcoin-cli -signet -rpcwallet="offline_wallet" finalizepsbt $processed_psbt | jq -r .hex >> ~/final_psbt.txt
+[offline]$ cat ~/final_psbt.txt
+0200000000010165e8a271a85542f2eda9f867e4dc40002e4f46b1b46640712fd0a77010d908040100000000fdffffff0280380100000000001600142da8e7cef3b9337f1e1309aef5d17ec47e66e5118326000000000000160014c9a498552f6fd359acfbbe57179a30b8b075b1c7024730440220659e544f37d1b48923df89dff71c63d996c0b233183c58f84a773d8e6306e5a00220765cb5a31403320ce49ab66d6fa097a70b00a131abfa832524bcc840af251f67012102b66303a69d660e98c49320afc28dd9abd8f34c9d59c4ca2148a243fa811ea6c800000000
+```
+
+### Broadcast the Finalized PSBT Transaction
+1. Broadcast the signed and finalized PSBT transaction (on the `final_psbt.txt` file) using the online wallet
+
+```sh
+[online]$ ./src/bitcoin-cli -signet sendrawtransaction $(cat ~/final_psbt.txt)
+e79497b11e3561cfd6349213ed71578d68411821d079855d56d419ba1db71170
+```
+
+### Confirm Wallet Balances
+
+1. Confirm the updated balance of the `online_receiving_wallet`.
```

Comment:
Would be able to drop this whole step if you eliminated the online_recieving_wallet.